### PR TITLE
Improved docs

### PR DESCRIPTION
**Pull request by v0lta**
TODO:
 - adopt the `sphinx_book_theme` (done)
 - add an introduction explaining how wavelets work ( in progress)
 - improve the function docstrings by adding appropriate formulae.
 - finish the 1D-swt code
     - see also: https://github.com/v0lta/PyTorch-Wavelet-Toolbox/issues/74 

The following terminal commands render the documentation:

``` bash
cd docs
make html
```
The docs depend on the `sphinx-book-theme` and `sphinxcontrib-bibtex` pip packages.

### CODE REVIEW

**Comment by v0lta** [comment]
@kgasenzer merci :-).

File: src/ptwt/conv_transform_2.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 The implementation relies on torch.nn.functional.conv2d and
 torch.nn.functional.conv_transpose2d under the hood.
 """
+# Written by the Pytorch wavelet toolbox team in 2024
```
Comment (cthoyt) [inline src/ptwt/conv_transform_2.py:6]: I don't think these are necessary, the license for the repo covers this

Comment (cthoyt) [reply src/ptwt/conv_transform_2.py:6]: (also, this makes the code more difficult to read)

Comment (v0lta) [reply src/ptwt/conv_transform_2.py]: I took these out.

**Comment by v0lta** [comment]
Docs update done.

**Review by Bendak09 — COMMENTED** [review]
I lost how to finish the transfer

**Pull request merged**